src/main/java/net/mcreator/blockly/java/blocks/CancelEventBlock.java

```diff
@@ -60,11 +60,11 @@
 				}
 			} else {
 				master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-						"This procedure does not use global trigger so the cancel event block can not cancel any trigger"));
+						L10N.t("blockly.errors.cancel_event.no_selected_trigger")));
 			}
 		} else {
 			master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
-					"Cancel event procedure block is not supported in this editor!"));
+					L10N.t("blockly.errors.unsupported", "blockly.block.cancel_event")));

```

Comment:
No need to have this in two strings

Reply:
I decided to make 2 strings because I didn't want to duplicate strings when all blocks will be externalozed. Several blocks use the first string, so instead to translate several times the sane strings, only the block's name is translated + a common string.

Reply:
Ok I see, right now it is only one so I thought it was not ok

Reply:
If at the end of the externalization, no other notes use it I will change it, but for the moment we can keep it because I'm pretty sure it will be useful.

Reply:
I think we should not do this too much, as this over-complicates things, where translators can simply copy the text and it is also easier for them to locate where the text is shown if it is not "assembled like this